README.md

```diff
@@ -1 +1,197 @@
+  - ( ) I'm familiar
+  - ( ) Practitioner
+  - ( ) Expert. Used in several scenarios, failed and learned from the mistakes, and being able to teach others.
+
+- How do you assess yourself on Tactical Design of DDD? (choose one)
+  - ( ) Never used
+  - ( ) I'm familiar with the concepts
+  - ( ) I'm familiar
+  - ( ) Practitioner
+  - ( ) Expert. Used in several scenarios, failed and learned from the mistakes, and being able to teach others.
+
+- Which sources have you used to learn about Domain-Driven Design? (Multiple choices)
+  - [ ] Book: Domain-Driven Design by Erik Evans
+  - [ ] Book: Implementing Domain-Driven Design by Vaughn Vernon
+  - [ ] Book: Domain-Driven Design Distilled by Vaughn Vernon
+  - [ ] Book: Learning Domain-Driven Design: Aligning Software Architecture and Business Strategy by Vladik Khononov
+  - [ ] Book: Patterns, Principles, and Practices of Domain-Driven Design by Scott Millett
+  - [ ] Book: Strategic Monoliths and Microservices by Vaughn Vernon
+  - [ ] Conferences
+  - [ ] Blogs and Online Resources
+  - [ ] Workshops and Trainings
+  - [ ] Others... (open text)
+
+### General DDD Concepts
+
+- Which is the most important thing about DDD? It's too open in purpose, there's no right and wrong answer. (Open Text)
+
+- What's a Domain Model? (Open Text)
+
+- What's Domain Modelling? (Open Text)
+
+- In which scenario you would use DDD? (Open Text)
+
+- In which scenario you would NOT use DDD? (Open Text)
+
+- When do you know what you're doing DDD wrong? (Open Text)
+
+- DDD helps you in (Multiple choices)
+  - [ ] Manage Business Complexity
+  - [ ] Mange Technical Complexity
+  - [ ] Define good Microservices
+  - [ ] Have good Architecture because of Tactical Patterns
+  - [ ] Other... (Open text)
+
+- What's a Big Ball of Mud? (Open text)
+
+### Strategic Design
+
+- Which Strategic Patterns are you familiar with? (Multiple choices)
+  - [ ] Bounded Contexts
+  - [ ] Ubiquitous Language
+  - [ ] Core Domain and Sub Domains
+  - [ ] Context Mapping
+  - [ ] Other... (Open text)
+
+- Which of those Patterns are used for the Problem Space? (Multiple choices)
+  - [ ] Bounded Contexts
+  - [ ] Ubiquitous Language
+  - [ ] Core Domain and Sub Domains
+  - [ ] Context Mapping
+  - [ ] Microservices
+  - [ ] Other... (Open text)
+
+- Which of those Patterns are used for the Solution Space? (Multiple choices)
+  - [ ] Bounded Contexts
+  - [ ] Ubiquitous Language
+  - [ ] Core Domain and Sub Domains
+  - [ ] Context Mapping
+  - [ ] Microservices
+  - [ ] Other... (Open text)
+
+- How do you identify a Bounded Context? (Open text)
+
+- How do you know what's Core Domain? (Open text)
+
+- How do you know what's Supportive Subdomain? (Open text)
+
+- How do you know what's Generic Subdomain? (Open text)
+
+- How do you differentiate between them? (Open text)
+
+- What's Context Mapping used for? (Open text)
+
+- Which kind of Context Mappings do you know? (Open text)
+
+### Domain Modelling with Event Storming
```

Comment:
This section seems quite useful as it's a bit more objective and reflecting on experiences.